docs/features/startup_and_waits.md

```diff
@@ -1,5 +1,13 @@
 # Waiting for containers to start or be ready
 
+## Depending on another container
```

Comment:
I think it would make sense to add it to the end of the page, **after** we talk about the _concept_ of waiting, as otherwise it is a bit too early